__mocks__/eslint.ts

```
@@ -0,0 +1,104 @@
+
+const mockLintTextSpy = jest.fn(mockLintText);
+
+export = Object.assign(eslint, {
+  ESLint: jest.fn(MockESLint),
+  mock: {
+    calculateConfigForFile: mockCalculateConfigForFileSpy,
+    lintText: mockLintTextSpy,
+  },
+});
+
+function MockESLint(options: ESLint_.Options): MockESLint {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MockESLint'. Consider to delete it or rename it.

'MockESLint' is defined here:


(lint/suspicious/noRedeclare)